xk6it/xk6it.go

```diff
@@ -0,0 +1,127 @@
+// Package xk6it contains integration test runner of the xk6 extensions.
+package xk6it
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"go.k6.io/xk6"
+	"golang.org/x/mod/modfile"
+)
+
+// Run builds k6 in temporary location and run scripts matching the given pattern.
+// For the given integration script, it searches for the containing go module
```

Comment:
What is "integration script" referring to here ❓ 

Also, just for my understanding is that this function is a way to programmatically `xk6 run` a xk6 extension, based on the assumption it's structured in a way that it has a dedicated go module (the actual structure is not important, but I just want to double check my understanding 👍🏻):
```
my_extension/
register.go
go.mod
my_module/
  module.go
```

Reply:
you are  right